### DISCUSSION TITLE
Short videos for KubeCon booth

### DISCUSSION THREAD

**Discussion by jpkrohling** [jaegertracing/jaeger#2808] [Ideas]
Folks, 

For KubeCon virtual events, there's the possibility of having small videos added to the booth. I recall Yuri mentioning in the past that it would be nice to have short videos demonstrating specific features, so, looks like we have good reasons to start recording them.

Here's an initial list of videos that I think we could be recording:

- 1-minute elevator pitch about the project
- 5-minute demo, possibly using Hot ROD, introducing the main concepts of distributed tracing
- a 10-minute presentation introducing the project
  - how Jaeger fits in the observability world
  - how Jaeger fits in a distributed tracing pipeline
  - main UI features
  - call to action to join our community

Do we have volunteers for those videos? I could give it a shot if no one else wants.

**Comment by jkowall** [comment]
My company might be able to help with these, we are discussing :)

**Reply by jpkrohling** [reply]
Do you have updates about this, @jkowall?

**Reply by jkowall** [reply]
@albertteoh did one already I believe. If we need more I can get our team to do more of them.
